napari/_qt/widgets/qt_progress_bar.py

```diff
@@ -11,6 +11,8 @@
 
 
 class ProgressBar(QWidget):
+    closed = QtCore.Signal()
```

Comment:
personally, I prefer to be relatively conservative about adding signals that seem kinda like they should have been there in the first place (there's almost always a "qt way")...  does [`pbar.destroyed`](https://doc.qt.io/qt-5/qobject.html#destroyed) not do what you need here?

if you definitely do need the event, probably best to emit from the [`closeEvent()`](https://doc.qt.io/qt-5/qwidget.html#closeEvent) method instead of the [`close()`](https://doc.qt.io/qt-5/qwidget.html#close) method

Reply:
Connecting to `destroyed` works perfectly as far as I can tell, however asserting that the indicator is not visible was failing (even though visually I can confirm it's not visible). I removed that assertion but I'm not sure whether there's a better way to test this